lib/epics/index.js

```diff
@@ -380,8 +469,17 @@ const submitApplicationsFromBundleFilesEpic = (action, state) => action.pipe(
         StatusUtils.submitJobStart(s, path.basename(bundleToUpload.bundlePath));
       }),
       mergeMap(bundle => StreamsRestUtils.artifact.uploadApplicationBundleToInstance(s, bundle.bundlePath).pipe(
+        tap((uploadRes) => {
```

Comment:
extractSab should be async and wait to return a new action; have separate epic to handle functionality that is currently below the delay(5000)